source/index.html

```diff
@@ -1,12 +1,206 @@
+        <button class="reviews__button-next" type="button">Следующий</button>
+      </div>
+      <a class="reviews__button button" href="#">Написать</a>
+    </section>
+
+    <section class="contacts">
+      <h2 class="contacts__title">Контакты</h2>
+      <dl class="contacts__list">
+        <dt class="contacts__item">e-mail:</dt>
+        <dd class="contacts__email">
+          <a href="mailto:info@mimimishkashop.ru">info@mimimishkashop.ru</a>
+        </dd>
+        <dt class="contacts__item">адрес:</dt>
+        <dd class="contacts__address">г. Санкт-Петербург ул. Большая Конюшенная д. 19/8, офис 101</dd>
+      </dl>
+
+      <div class="contacts__map">
+        <img class="contacts__map-image" src="img/content/map-mobile.jpg" alt="">
+      </div>
+      <a class="contacts__button button" href="#">Напишите нам</a>
+    </section>
+  </main>
+
+  <footer class="page-footer">
+    <ul class="social-list">
+      <li class="social-list__item">
+        <a class="social-list__link social-list__link--instagram" href="https://www.instagram.com/htmlacademy">Мы в
+          Инстаграме</a>
+      </li>
+      <li class="social-list__item">
+        <a class="social-list__link social-list__link--facebook" href="https://www.facebook.com/htmlacademy">Мы в
+          Фейсбуке</a>
+      </li>
+      <li class="social-list__item">
+        <a class="social-list__link social-list__link--twitter" href="https://twitter.com/htmlacademy_ru">Мы в
+          Твиттере</a>
+      </li>
+    </ul>
+
+    <div class="page-footer__copyright copyright">
+      <p class="copyright__text">Разработано:</p>
+      <a class="copyright__link" href="https://htmlacademy.ru/intensive/adaptive">
+        <svg class="copyright__logo" width="27" height="35" viewBox="0 0 27 35" fill="none" aria-labelledby="title">
+          <title id="title">Перейти на страницу Онлайн‑курса HTML и CSS. Адаптивная вёрстка и автоматизация.</title>
+          <path
+            d="M13.6 0H13.5L0 1.4V26.1L13.5 34.1L26.9 26.1V1.4L13.6 0ZM25 12.1L13.5 5.3V5.2H13.4L13.3 5.1V5.2L1.9 12.1V3.1L13.4 1.9L25 3.1V12.1ZM13.4 6.8L24.9 13.5L20.4 16.1L13.3 11.9V13.3L19.2 16.8L18.3 17.3L13.3 14.3V15.7L17.1 18L16.3 18.5L13.3 16.7V18.1L15.2 19.2L13.4 20.3L2 13.6L13.4 6.8ZM1.9 15.1L13.3 21.9V22.9L5.4 18.2V19.6L13.4 24.4V25.5L5.4 20.8V22.2L13.4 27L21.5 22.1V16.9L24.9 14.9V24.9L13.4 31.8L1.9 25V15.1Z"
+            fill="#63D1BB" />
+        </svg>
+      </a>
+    </div>
+  </footer>
+
+  <section class="modal">
+    <h2 class="modal__title">Добавить в корзину</h2>
+    <form class="size-table" method="get" action="https://echo.htmlacademy.ru">
+      <fieldset>
+        <legend class="size-table__legend">Выберите размер:</legend>
+        <ul class="size-list">
+          <li class="size-list__item">
+            <input class="size-list__input" id="size-s" type="radio" name="item-size" value="s">
+            <label class="size-list__label" for="size-s">S</label>
```

Comment:
```suggestion
            <input class="size-table__input" id="size-s" type="radio" name="item-size" value="s">
            <label class="size-table__label" for="size-s">S</label>
```

наговорю тебе на телегу голосом, а то писать долго